ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/OptimizeService.java

```diff
@@ -97,16 +91,15 @@ public OptimizeService() {
   }
 
   private void init() {
-    tablesLock.writeLock().lock();
     try {
       LOG.info("OptimizeService init...");
-      loadTables();
-      initOptimizeTasksIntoOptimizeQueue();
+      new Thread(() -> {
```

Comment:
These operations are time-consuming. Since this logic is executed when a new OptimizeService object is created, serial operation will affect ams startup

Reply:
sgtm